### PR TITLE
Fix a few lint warnings.

### PR DESCRIPTION
**Pull request by Dominator008**

### CODE REVIEW

File: closure/goog/testing/performancetimer.js [hunk 1]
```diff
@@ -194,6 +194,7 @@ goog.testing.PerformanceTimer.prototype.runTask = function(task) {
  * Finishes the run of a task by creating a result object from samples, in the
  * format described in {@code run}.
  * @see goog.testing.PerformanceTimer#run
+ * @param {Array<number>} samples The samples to analyze.
```
Comment (shicks) [inline closure/goog/testing/performancetimer.js]: We should add a lint warning about implicit nullability...

Comment (Dominator008) [inline closure/goog/testing/performancetimer.js]: @shicks That's part of analyzerChecks which is off for Closure Library now: https://github.com/google/closure-compiler/blob/master/src/com/google/javascript/jscomp/DiagnosticGroups.java#L507.

File: closure/goog/testing/propertyreplacer.js [hunk 2]
```diff
@@ -116,6 +116,7 @@ goog.testing.PropertyReplacer.hasKey_ = function(obj, key) {
     //   var dummy = document.body.constructor.prototype.className
     //   [Exception... "Cannot modify properties of a WrappedNative"]
     // </pre>
+    /** @suppress {unusedLocalVariables} */
```
Comment (shicks) [inline closure/goog/testing/propertyreplacer.js]: Internal linter currently complains about this line, saying "@suppress is not allowed here".  Is that changing?  Can we land the linter change first?

Comment (Dominator008) [inline closure/goog/testing/propertyreplacer.js]: I removed that suppression for now. In fact most of the lint warnings in Closure Library are "@suppress is not allowed here". I think we need to allow more suppressions in the Compiler: https://github.com/google/closure-compiler/issues/1615.

Comment (shicks) [inline closure/goog/testing/propertyreplacer.js]: Alright, sounds good - we can fix those up once they're allowed.

**Pull request closed without merging**